Dashboard/Models/RawData.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Dashboard.Models;
+
+public record RawData
+{
+    public string CarId { get; init; }
```

Comment:
U can fix this warning by adjusting all (or just hte ones that warn) to look like
```suggestion
    public required string CarId { get; init; }
```

Reply:
just means when the constructor or init statement is exited, it has to have a value

Reply:
Can do.

Reply:
https://github.com/nickowen44/AV-Datalogger/pull/10/commits/af9d6f189027cdae3bff3d19679e23588c2190d4